tests/models/falcon/test_modeling_falcon.py

```diff
@@ -461,17 +461,21 @@ def test_model_rope_scaling(self):
 
         # Inputs
         x = torch.randn(1, dtype=torch.float32, device=torch_device)  # used exlusively to get the dtype and the device
+        position_ids_short = torch.arange(short_input_length, dtype=torch.long, device=torch_device)
```

Comment:
(FYI: added a note to myself to move `test_model_rope_scaling` into a mixin :) )